src/pudl/helpers.py

```diff
@@ -972,11 +987,9 @@ def convert_cols_dtypes(df, data_source, name=None):
         if df.utility_id_eia.dtypes is np.dtype('object'):
             df = df.astype({'utility_id_eia': 'float'})
     df = (
-        df.replace(to_replace="<NA>", value={
-                   col: pd.NA for col in string_cols})
-        .replace(to_replace="nan", value={col: pd.NA for col in string_cols})
-        .astype(non_bool_cols)
+        df.astype(non_bool_cols)
```

Comment:
Can you be absolutely positive that that the string "<NA>" never shows up? Did you remove it on purpose? Isn't it what we'd expect if a StringDtype column were accidentally converted to a Python object based string column?

Reply:
are you referring to the first deleted replace here?

Reply:
Yes In the old code it looks like it was replacing both "<NA>" and "nan" with `pd.NA` but in the new code it only replaces "nan" with `pd.NA` right?

Reply:
yes, you are right here, I've added it back in.